app/models/condition.rb

```diff
@@ -88,16 +88,7 @@ def self.evaluate(cond, rec, inputs = {}, attr = :expression)
   end
 
   def self.do_eval(expr)
-    # Remove this [DEPRECATION] in the next version
```

Comment:
That actually was left behind when I reinstated the code after Bill removed it. He had it raising an error there but I did it in the expression class.